Watchman.Discord/Integration/DevscordFramework/CustomCommandsLoader.cs

```diff
@@ -39,9 +39,9 @@ public async Task InitDefaultCustomCommands()
             {
                 customCommands.AddRange(new List<AddCustomCommandsCommand>
                 {
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<GetRoles>[\w\W\s\""]*)", serverId),
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s*(?<GetRoles>[\w\W\s\""]*)", serverId),
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s*(?<GetRoles>[\w\W\s\""]*)\s*-((?<Safe>safe)|(?<Unsafe>unsafe))", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s*(?<Roles>[\w\W\s\""]*)\s*-((?<Safe>safe)|(?<Unsafe>unsafe))", serverId),
```

Comment:
tak naprawdę to można prościej zamiast tych \w\W\s itd. dać kropkę. Kiedyś na ogólnym była o tym mowa, że to jest to samo.